src/examples/generation.ts

```diff
@@ -91,7 +95,12 @@ treeGen
         instance.addDetail({ component: 'maybeBranch', at: root });
     })
     .define('leaf', (root: Point, instance: GeneratorInstance) => {
-        const leaf = instance.add(root.attach(leafSphere));
+        const clonedModel = leafModel.cloneDeep();
+        const leaf = clonedModel.root();
+        const nodesToAdd = root.attachModel(leaf, clonedModel.nodes);
+        nodesToAdd.forEach((node: Node) => {
+            instance.add(node);
```

Comment:
I'm going to make a separate issue to figure out a better way to deal with this, because `instance.add` is super easy to forget. (Not a comment on this PR, just another bit of polish we should add in general)